Inc/ssd1306/ssd1306_tests.h

```diff
@@ -0,0 +1,22 @@
+#ifndef __SSD1306_TEST_H__
```

Comment:
Do you think it is worth it to compile in test functions?

Reply:
The display test functions aren't compiled in as there are no references to them in the code, but they come by default with the SSD1306 library I used. I'm going to add a menu system to let you configure run settings etc, and I thought it might be useful to have a "display test" function as part of the menu, but I can get rid of them in the meantime if you don't want unused stuff lying around.